src/commands/run.ts

```diff
@@ -2,6 +2,42 @@ import { startServer } from '../server.js';
 import trackEvent from '../utils/analytics.js';
 import { log, logError, logInfo } from '../utils/logger.js';
 import * as os from 'os';
+import type { HttpServerOptions } from '../utils/types.js';
+
+// Function to parse command line arguments
+function parseArgs(): { useHttpTransport: boolean; httpOptions: HttpServerOptions } {
```

Comment:
FYI: we now are using commander so we should push this into the index.ts file. But see my other comment about HTTP transport.